packages/@aws-cdk/aws-lambda/lib/pipeline-action.ts

```diff
@@ -9,6 +9,22 @@ import { FunctionRef } from './lambda-ref';
  * or through {@link FunctionRef#addToPipeline}.
  */
 export interface CommonPipelineInvokeActionProps extends codepipeline.CommonActionProps {
+  /**
+   * The optional input Artifacts of the Action.
```

Comment:
Maybe provide some pointers as to how to read the input artifacts from the lambda code and how to produce the output artifacts

Reply:
Done.